pkg/cvo/sync_worker.go

```diff
@@ -148,11 +148,16 @@ type SyncWorker struct {
 
 	// updated by the run method only
 	payload *payload.Update
+
+	// exclude is an identifier used to determine which
+	// manifests should be excluded based on an annotation
+	// of the form exclude.release.openshift.io/<identifier>=true
+	exclude string
```

Comment:
Do we want a single string, here or a `[]string` of identifiers to exclude?  Related to [this thread][1].

[1]: https://github.com/openshift/cluster-version-operator/pull/269#discussion_r346049977

Reply:
@wking the idea is that the identifier would be the equivalent of a single profile. If you have a different exclusion profile, you should use a different identifier. Manifests in the individual operators can contain exclusion annotations for multiple profiles.